cordova-lib/spec-cordova/save.spec.js

```diff
@@ -0,0 +1,435 @@
+                done();
+            });
+        }, timeout);
+
+        it('spec.15 cordova plugin add git_url --save', function (done) {
+            platform('add', platformLocalPath)
+            .then(function () {
+                return cordova.raw.plugin('add', pluginGitUrl, { 'save': true });
+            }).then(function () {
+                expect(helpers.getPluginSpec(appPath, pluginName)).toBe(pluginGitUrl);
+                done();
+            }).catch(function (err) {
+                expect(true).toBe(false);
+                console.log(err.message);
+                done();
+            });
+        }, timeout);
+
+        it('spec.16 cordova plugin add local_folder --save', function (done) {
+            platform('add', platformLocalPath)
+            .then(function () {
+                return cordova.raw.plugin('add', localPluginPath, { 'save': true });
+            }).then(function () {
+                expect(helpers.getPluginSpec(appPath, localPluginName)).toBe(localPluginPath);
+                done();
+            }).catch(function (err) {
+                expect(true).toBe(false);
+                console.log(err.message);
+                done();
+            });
+        }, timeout);
+    });
+
+    describe('(plugin remove)', function () {
+        it('spec.17 cordova plugin remove plugin-id --save [config.xml is empty]', function (done) {
+            platform('add', platformLocalPath)
+            .then(function () {
+                return cordova.raw.plugin('add', pluginName);
+            }).then(function () {
+                return cordova.raw.plugin('rm', pluginName, { 'save': true });
+            }).then(function () {
+                expect(helpers.getPluginSpec(appPath, pluginName)).toBe(null);
+                done();
+            }).catch(function (err) {
+                expect(true).toBe(false);
+                console.log(err.message);
+                done();
+            });
+        }, timeout);
+
+        it('spec.18 cordova plugin remove plugin-id --save [config.xml contains git-url of plugin]', function (done) {
+            platform('add', platformLocalPath)
+            .then(function () {
+                return cordova.raw.plugin('add', pluginName);
+            }).then(function () {
+                helpers.setPluginSpec(appPath, pluginName, pluginGitUrl);
+                return cordova.raw.plugin('rm', pluginName, { 'save': true });
+            }).then(function () {
+                expect(helpers.getPluginSpec(appPath, pluginName)).toBe(null);
+                done();
+            }).catch(function (err) {
+                expect(true).toBe(false);
+                console.log(err.message);
+                done();
+            });
+        }, timeout);
+    });
+
+    describe('plugin update', function () {
+        //plugin update functionality is not yet implemented in cordova-cli
+        xit('spec.19 cordova plugin update cordova-plugin-console --save', function (done) {
```

Comment:
That's because there's no 'cordova plugin update' functionality. Please, delete those tests.